units/event_metajob.go

```diff
@@ -159,21 +159,6 @@ func (j *eventMetaJob) dispatchLoop(ctx context.Context) error {
 			}))
```

Comment:
Left in the logging for when mgo returns an error trying to insert the notification.
(Can't comment on line 153 :disappointed:)